src/main/webapp/app/shared/circular-progress-bar/circular-progress-bar.component.scss

```diff
@@ -125,22 +125,22 @@ Due to the split circle of progress-left and progress right, we must use the ani
         }
     }
     //animation only the right side if above 50%
-    @if $i > ($howManySteps/2) {
+    @if $i > math.div($howManySteps, 2) {
         .progress-circle[data-percentage='#{$stepName}'] {
             .progress-right .progress-bar-circle {
-                animation: loading-#{$howManySteps/2} $animationTime linear forwards; //set the animation to longest animation
+                animation: loading-#{$howManySteps, 2} $animationTime linear forwards; //set the animation to longest animation
```

Comment:
Isn't here the `div` function call missing?

Reply:
Indeed, good catch. Fixed, please recheck.